### PR TITLE
Clean up ecovacs documentation

### PR DESCRIPTION
**Pull request by edenhaus**
## Proposed change

Removed outdated documentation about ecovacs.
HA has a built-in functionality to get the logs from an integration.
The vacuum service calls are documented on the vacuum doc page.

Also the ecovacs integration used now two libraries and therefore removed pointing to only one of it.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108167
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @edenhaus :+1:

File: source/_integrations/ecovacs.markdown [hunk 1]
```diff
@@ -23,90 +24,15 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
-Note: For some countries, you will need to set `continent` to `ww` (meaning worldwide.) There is unfortunately no way to know the correct settings other than guessing and checking. See the [py-sucks library protocol documentation](https://github.com/mib1185/py-sucks/blob/master/protocol.md) for more information about what has been figured out about the Ecovacs servers.
+Note: For some countries, you will need to set `continent` to `ww` (meaning worldwide). There is unfortunately no way to know the correct settings other than guessing and checking.
```
Comment (MartinHjelmare) [inline source/_integrations/ecovacs.markdown:27]: Shouldn't we remove the mention of continent?

Comment (edenhaus) [reply source/_integrations/ecovacs.markdown:27]: Yes will do it in a follow up

Comment (edenhaus) [reply source/_integrations/ecovacs.markdown:27]: https://github.com/home-assistant/home-assistant.io/pull/30959

**Pull request merged**